python/statsforecast/models.py

```diff
@@ -2129,38 +2129,62 @@ def __init__(
         )
 
 # %% ../../nbs/src/core/models.ipynb 129
-@njit(nogil=NOGIL, cache=CACHE)
-def _ses_fcst_mse(x: np.ndarray, alpha: float) -> Tuple[float, float, np.ndarray]:
-    r"""Perform simple exponential smoothing on a series.
+@njit(nogil=NOGIL, cache=CACHE, fastmath=True)
+def _ses_sse(alpha: float, x: np.ndarray) -> float:
```

Comment:
Is this really worth it? Both of the functions you extracted are basically the same, this just avoids filling the fitted values but I doubt that makes a huge difference.

Reply:
I think we can keep this function and change the other one.